### ISSUE TITLE
Support for relative positioning/auto layout

### ISSUE THREAD

**Issue by pcolton** [Pixate/pixate-freestyle-ios#66]

**Comment by daronjay** [comment]
Yes, Autolayout integration is the missing piece from the puzzle. What about at least giving a roadmap so developers considering Pixate will be encouraged to hear that you are planning to "address" the issue?
